change/@fluentui-react-native-experimental-radio-group-346f9bef-d345-48b5-b108-de0a1146e8fd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Changelog has nothing to do with the actual change. Please update to include more accurate message.